nss-day-cohort-44/hello-world-the-jellyfish-chain
# Jk add aside

# Description
Please check my html for aside and footer looks perfect and workable and that the dummy text is working for all the parts. Approve working skeleton.
## Type of change
- [x ] New feature (non-breaking change which adds functionality)
functionality to not work as expected)
# Testing Instructions
1. `git fetch --all`
1. `git checkout jk-add-aside`
1. `serve`
1. check html for aside and footer, look for filler text, approve working skeleton.
